src/Output.cxx

```diff
@@ -1422,6 +1425,12 @@ void ASTVisitor::PrintBefriendingAttribute(clang::CXXRecordDecl const* dx)
 }
 
 //----------------------------------------------------------------------------
+void ASTVisitor::PrintElaboratedAttribute(clang::QualType t)
+{
+  this->OS << " elaborated=\"" << t->isElaboratedTypeSpecifier() << "\"";
```

Comment:
Since these are rare and gccxml didn't produce this output I'd prefer to use `elaborated="1"` for true and nothing for false.